src/wayland/seat/mod.rs

```diff
@@ -186,6 +186,19 @@ impl<D: SeatHandler + 'static> Seat<D> {
             .map(|arc| Self { arc })
     }
 
+    /// Retrieves [`WlSeat`](wl_seat::WlSeat) resources for a given client
+    pub fn resources_for_client(&self, client: &ClientId) -> Vec<WlSeat> {
```

Comment:
We probably should be consistent with Output, which provides a with_client_outputs function that takes a closure.

Reply:
I aggree, not saing that one signature or name is better than the other, just that it would be nice to have them be similar.

Reply:
@vberger pointed out that ClientId was not meant to be used directly when I wanted to change Output to take a ClientId. With this [commit](https://github.com/Smithay/wayland-rs/commit/13f6a9beb05eabac56aa372e1d38b12bfa987982) (release pending) this should also be no longer necessary.

Reply:
k, changing it again